docs/basics/integrations/deque.md

```diff
@@ -30,23 +31,23 @@ Check out the [Deque Accessibility Guide](https://www.deque.com/web-accessibilit
 
 ## Sauce Labs UI
 
-Sauce Labs will display the accessibility results in our UI as a tab in the Job itself:
+Sauce Labs will display the accessibility results in our UI as a tab in the job itself:
 
 <img src={useBaseUrl('img/accessibility/example.png')} alt="Accessibility Example in Sauce UI"/>
 
 
 ## Language Specific Examples
 
 <Tabs
-defaultValue="java"
-values={[
-{ label: 'Java', value: 'java', },
-{ label: 'WebdriverIO', value: 'wdio', },
-{ label: 'Python', value: 'python', },
-{ label: 'Ruby', value: 'ruby', },
-{ label: 'C#', value: 'dotnet', }
-]
-}>
+  groupId="axe-lang-examples"
+  defaultValue="java"
+  values={[
+    {label: 'Java', value: 'java'},
+    {label: 'WebdriverIO', value: 'webdriver'},
```

Comment:
```suggestion
    {label: 'JavaScript', value: 'js'},
```

Reply:
Either way `webdriver` is confusing because WebdriverIO is not abbreviated as WebDriver. Better to keep them all as languages.